eZ/Publish/Core/settings/repository/siteaccessaware.yml

```diff
@@ -1,5 +1,8 @@
 parameters:
     languages: []
+    storage_dir: var/ezdemo_site/storage
+    legacy_url_prefix: var/ezdemo_site/storage
+    url_prefix: var/ezdemo_site/storage
```

Comment:
Hmm, I think that `ezdemo_site` is a eZ Publish left-over. It should be safe to change it to `site`, because AFAIR those settings are overridden on either bundle or meta level anyway (correct me if I'm wrong).

Reply:
This would be nice but needs changes in integration tests (ImageIntegrationTest, MediaIntegrationTest.php, FileBaseIntegrationTest, BinaryFileIntegrationTest...) Also, we have `storage_dir: var/ezdemo_site/storage` in eZ/Publish/Core/settings/settings.yml:9. I think that cleaning this up is out of scope.